data/json/items/armor/coats.json

```diff
@@ -1600,6 +1649,146 @@
+    "type": "ARMOR",
+    "name": { "str": "cheap ghost's robe" },
+    "description": "A cheap ghost's robe made with a white sheet and with some cuts for the eyes.  It uses a simple hat for keeping the costume in place.",
+    "weight": "880 g",
+    "volume": "550 ml",
+    "price": 2000,
+    "price_postapoc": 60,
+    "to_hit": -1,
+    "material": [ "cotton" ],
+    "symbol": "[",
+    "looks_like": "sheet",
+    "color": "white",
+    "armor": [
+      { "covers": [ "head" ], "coverage": 100, "encumbrance": 10 },
+      { "covers": [ "mouth" ], "coverage": 100, "encumbrance": 5 },
+      { "covers": [ "torso" ], "coverage": 100, "encumbrance": 2 },
+      { "covers": [ "arm_l", "arm_r" ], "coverage": 85, "encumbrance": 2 },
+      { "covers": [ "leg_l", "leg_r" ], "coverage": 80 },
+      { "covers": [ "eyes" ], "coverage": 5, "encumbrance": 1 }
+    ],
+    "warmth": 5,
+    "material_thickness": 0.1,
+    "flags": [ "OVERSIZE", "OUTER", "ALLOWS_NATURAL_ATTACKS" ]
+  },
+  {
+    "id": "grim_reaper_robe",
+    "repairs_like": "cloak",
+    "type": "ARMOR",
+    "name": { "str": "grim reaper's robe" },
+    "description": "A black hooded grim reaper's robe with a face cover to hide the lack of facial skin of the user.  It combines nicely with a scythe.  Use it to remove the face cover and show yourself before your victims.",
+    "weight": "453 g",
+    "volume": "550 ml",
+    "price": 3500,
+    "price_postapoc": 90,
+    "to_hit": -1,
+    "material": [ "cotton" ],
+    "symbol": "[",
+    "looks_like": "cloak",
+    "color": "dark_gray",
+    "armor": [
+      { "covers": [ "head" ], "coverage": 100, "encumbrance": 4 },
+      { "covers": [ "mouth", "eyes" ], "coverage": 100, "encumbrance": 5 },
+      { "covers": [ "torso" ], "coverage": 100, "encumbrance": 2 },
+      { "covers": [ "arm_l", "arm_r", "leg_l", "leg_r" ], "coverage": 95, "encumbrance": 2 }
+    ],
+    "use_action": {
+      "type": "transform",
+      "msg": "The darkness surrounding your face lightens.",
+      "target": "grim_reaper_robe_faceless",
+      "menu_text": "Remove face cover"
+    },
+    "warmth": 10,
+    "material_thickness": 0.1,
+    "flags": [ "OVERSIZE", "OUTER", "ALLOWS_NATURAL_ATTACKS" ]
+  },
+  {
+    "id": "grim_reaper_robe_faceless",
+    "copy-from": "grim_reaper_robe",
+    "repairs_like": "cloak",
+    "type": "ARMOR",
+    "name": { "str": "grim reaper's robe (faceless)", "str_pl": "grim reaper's robes (faceless)" },
+    "description": "A black hooded grim reaper's robe with the face cover removed to show the lack of facial skin of the user.  It combines nicely with a scythe.  Use it to adjust the face cover in place and hide your lack of facial features.",
+    "use_action": {
+      "type": "transform",
+      "msg": "You shroud your face in darkness.",
+      "target": "grim_reaper_robe",
+      "menu_text": "Adjust face cover"
+    },
+    "armor": [
+      { "covers": [ "head" ], "coverage": 100, "encumbrance": 4 },
```

Comment:
I feel like I often get confused about transforms but I think these two might be backwards or mixed up?

Reply:
Oh no, these are fine, the "faceless" thing in the name is because of the "face cover"less, but with a darker sounding name (And if you were a real grim reaper, you would really be faceless if you remove the face cover). Maybe I should add a comment about it.